extensions/cast/src/main/java/com/google/android/exoplayer2/ext/cast/CastPlayer.java

```diff
@@ -976,6 +997,15 @@ private boolean updateTimelineAndNotifyIfChanged() {
                     getCurrentMediaItem(), MEDIA_ITEM_TRANSITION_REASON_PLAYLIST_CHANGED));
       }
       updateAvailableCommandsAndNotifyIfChanged();
+
+      if (playingPeriodChanged) {
+        // If the new source is a live stream enable live stream status updates.
+        refreshLiveStreamStatusHandler.removeCallbacksAndMessages(null);
+        if (remoteMediaClient != null && remoteMediaClient.isLiveStream()) {
```

Comment:
I tested this with a couple of live streams but I never got `remoteMediaClient.isLiveStream()` return true. It is always false so it does not start polling. 

What live streams are you using when this returns true? HLS or DASH?

Reply:
Ok. I was planning to create a question issue for this but here it comes - you have to remove the following line from 
from DefaultMediaItemConverter.toMediaQueueItem():
```
.setStreamType(MediaInfo.STREAM_TYPE_BUFFERED)
```

The Cast receiver behaves properly with both types of streams this way.